docs/source/usage/inventory.rst

```diff
@@ -160,6 +197,11 @@ FortiClient EMS Devices
 
   The password used to login to the FortiClient EMS.
 
+**ssl_verify** *bool* (optional, default: true)
+
+  Check hosts SSL certificate (true) or not (false). Please be aware that disabling SSL certificate
+  verification is a security risk and should not be used in a production environment.
```

Comment:
Perhaps we should add the option to provide a custom CA or certificate hash that may be accpeted

Reply:
Opened issue #56 to resolve this.